setup.py

```diff
@@ -23,7 +23,7 @@
     "ctapipe-stage1 = ctapipe.tools.stage1:main",
     "ctapipe-merge = ctapipe.tools.dl1_merge:main",
 ]
-tests_require = ["pytest", "pandas>=0.24.0"]
+tests_require = ["pytest", "pandas>=0.24.0", "importlib_resources"]
```

Comment:
Only for `python < 3.9`, there is a special syntax for this: `importlib_resources;python_version<'3.9'`

Reply:
thanks, done